Documentation/migration-scripts/python-scripts/fetching_and_auto_mapping_groups.py

```
@@ -0,0 +1,287 @@
+    return res.json()["id"]
+
+
+def get_groups(url, session_id):
+    return requests.get(
+        f"{url}/permissions/group",
+        headers={"X-Metabase-Session": session_id}
+    ).json()
+
+
+def get_users(url, session_id):
+    return requests.get(
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)